src/main/java/core/basesyntax/model/RegistrationServiceImpl.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.model;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exception.RegistrationException;
+import core.basesyntax.service.RegistrationService;
+
+public class RegistrationServiceImpl implements RegistrationService {
+    static final int AGE_PERMISION = 18;
```

Comment:
```suggestion
    private static final int MIN_AGE = 18;
```

Reply:
Not neccessary to change the name of the variable(but consider to, since age permission sound like smth different from min age), but you should change to private